com.unity.ml-agents/Runtime/SideChannel/FloatPropertiesChannel.cs

```diff
@@ -42,9 +42,21 @@ public class FloatPropertiesChannel : SideChannel, IFloatProperties
         Dictionary<string, float> m_FloatProperties = new Dictionary<string, float>();
         Dictionary<string, Action<float>> m_RegisteredActions = new Dictionary<string, Action<float>>();
 
-        public override int ChannelType()
+        public FloatPropertiesChannel(int channelId = -1)
         {
-            return (int)SideChannelType.FloatProperties;
+            if (channelId == -1)
```

Comment:
nit: could `-1` be made a public constant, something like `kSideChannelIdUnset`, or `ReservedChannelId.Unset`